### PR TITLE
Update locked contract instructions

### PR DESCRIPTION
**Pull request by ipopescu**
### Related links

While working on presenting the contract upgrade material (card [#16](https://github.com/casperlabs/proservices_general/issues/16)), I noticed that a few sections needed updates.

### Changes

- Mention new_locked_contract instead of create_contract_package_at_hash
- Mentioned call_versioned_contract
- Added a new section for disabling a contract version

### Notes

Ran locally without issues.
Screen Shot 2022-05-26 at 2 43 32 PM

### CODE REVIEW

File: source/docs/casper/dapp-dev-guide/tutorials/upgrade-contract.md [hunk 1]
```diff
@@ -103,7 +103,9 @@ make build-contract
 
 ### Step 5. Verify your changes 
 
-You can write unit tests to verify the behavior of the new contract version. For the simple example counter above, here are the [corresponding tests](https://github.com/casper-network/casper-node/blob/dev/smart_contracts/contracts/test/contract-context/src/main.rs). Notice how the tests store and verify the [contract's version](https://github.com/casper-network/casper-node/blob/8356f393d361832b18fee7227b5dcd65e29db768/smart_contracts/contracts/test/contract-context/src/main.rs#L172-L173).
+You can write unit tests to verify the behavior of the new contract version. Use the [call_versioned_contract](https://docs.rs/casper-contract/latest/casper_contract/contract_api/runtime/fn.call_versioned_contract.html) function.
```
Comment (darthsiroftardis) [inline source/docs/casper/dapp-dev-guide/tutorials/upgrade-contract.md:106]: So for every new contract added to the package (which also ups the highest enabled version), you'd obtain a new contract hash as well. So you can use either the contract hash directly, which is the primary identifier of the contract. Alternatively, you can use `call_versioned_contract` specifying the newly added version, however, its not the only way

**Review by darthsiroftardis — COMMENTED** [review]
One comment, but otherwise content looks good

**Comment by ipopescu** [comment]
Requesting a re-review for this small PR, @darthsiroftardis or @ACStoneCL. Thanks!

**Review by ACStoneCL — APPROVED** [review]
Looks good to me!

**Comment by ipopescu** [comment]
casperstats.io is offline breaking the build. Merging this PR since everything else is working.

**Pull request merged**